backend/de.metas.business/src/main/java/de/metas/invoice/service/impl/AbstractInvoiceBL.java

```diff
@@ -163,7 +163,7 @@ public final I_C_Invoice creditInvoice(@NonNull final I_C_Invoice invoice, final
 		Check.errorIf(isCreditMemo(invoice), "Param 'invoice'={} may not be a credit memo");
 		Check.assume(invoice.getGrandTotal().signum() != 0, "GrandTotal!=0 for {}", invoice);
 
-		if (creditCtx.isCreditedInvoiceReinvoicable())
+		if (creditCtx.isReferenceInvoice())
 		{
 			Check.errorIf(invoice.isPaid(), "Param 'invoice'={} may not yet be paid");
```

Comment:
please change this error a bit. e.g. throw an AdempiereException with text 
"If the credit memo shall reference the credited invoice, then that invoice may not yet be paid");
and append the C_Invoice_ID as exception parameter